www/assets/js/lib/util.ts

```diff
@@ -33,3 +35,20 @@ export const sentryCaptureException = (excetpion: any) => {
 export const sentryCaptureMessage = (message: string) => {
   Sentry.captureException(message)
 }
+
+/**
+ * Converts course list array into learning resources array while maintaing order
+ */
+export const getLearningResourcesFromCourseList = (
```

Comment:
What does this PR have to do with resources?

Reply:
This doesn't refer to resources as we define them nextgen OCW, rather this refers to `LearningResource` objects as they're defined in `open-discussions` because that's what's being returned by the Open search API.

Reply:
We are converting course JSON to learning resource for every course in a particular course list [here](https://github.com/mitodl/ocw-hugo-themes/blob/96eb72e6ef08cbf8188c7b9189c4c826864368e5/www/assets/js/hooks/hugo_data.ts#L91), which is then rendered by the course list component. 
I have wrapped up that logic of conversion (courselist -> array of learning resources) in a `util` to increase reusability.